scripts/config/standard/kyverno.yaml

```
@@ -20,6 +20,8 @@ backgroundController:
             - nodes
             - nodes/status
             - pods
+          verbs:
+            - '*'
```

```suggestion
            - create
            - update
            - patch
            - delete
            - get
            - list
```

No need to provide more than before.